test/tools/unified-spec-runner/operations.ts

```diff
@@ -659,6 +672,42 @@ operations.set('runCommand', async ({ entities, operation }: OperationFunctionPa
   return db.command(command, options);
 });
 
+operations.set('runCursorCommand', async ({ entities, operation }: OperationFunctionParams) => {
+  const db = entities.getEntity('db', operation.object);
+  const { command, ...opts } = operation.arguments!;
+  const cursor = db.runCursorCommand(command, {
+    readPreference: ReadPreference.fromOptions({ readPreference: opts.readPreference }),
+    session: opts.session
+  });
+
+  if (!Number.isNaN(+opts.batchSize)) cursor.setBatchSize(+opts.batchSize);
+  if (!Number.isNaN(+opts.maxTimeMS)) cursor.setMaxTimeMS(+opts.maxTimeMS);
+  if (opts.comment !== undefined) cursor.setComment(opts.comment);
+
+  return cursor.toArray();
+});
+
+operations.set('createCommandCursor', async ({ entities, operation }: OperationFunctionParams) => {
+  const collection = entities.getEntity('db', operation.object);
+  const { command, ...opts } = operation.arguments!;
+  const cursor = collection.runCursorCommand(command, {
+    readPreference: ReadPreference.fromOptions({ readPreference: opts.readPreference }),
+    session: opts.session
+  });
+
+  if (!Number.isNaN(+opts.batchSize)) cursor.setBatchSize(+opts.batchSize);
+  if (!Number.isNaN(+opts.maxTimeMS)) cursor.setMaxTimeMS(+opts.maxTimeMS);
+  if (opts.comment !== undefined) cursor.setComment(opts.comment);
+
+  // The spec dictates that we create the cursor and force the find command
+  // to execute, but the first document must still be returned for the first iteration.
+  const result = await cursor.tryNext();
```

Comment:
Why not just call `initialize` like our change stream helper does for unified change stream tests?